### PR TITLE
Add new multi module project

### PR DESCRIPTION
**Pull request by bennetelli**

### CODE REVIEW

**Review by gwenneg — COMMENTED** [review]
The `.github` folder should stay at the repository root level.

**Comment by theute** [comment]
It doesn't seem to be a `git mv` but rather a new directory within the existing with duplicate content ?

File: notifications-backend/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```
Comment (gwenneg) [inline notifications-backend/LICENSE.txt:2]: This file should stay at the repository root level.

File: notifications-backend/mvnw [hunk 2]
```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```
Comment (gwenneg) [inline notifications-backend/mvnw:1]: This file should stay at the repository root level.

File: notifications-backend/mvnw.cmd [hunk 3]
```diff
@@ -0,0 +1,182 @@
+@REM ----------------------------------------------------------------------------
```
Comment (gwenneg) [inline notifications-backend/mvnw.cmd:1]: This file should stay at the repository root level.

**Comment by bennetelli** [comment]
@theute I didn't know that the git mv feature exist :(

**Comment by theute** [comment]
Git is smart enough that git add / git rm is equivalent (and then also keeps history), but I don't see the rm in the changelog.
Now in this PR the whole history is lost since the old files are still in `src`

**Comment by bennetelli** [comment]
Don't know what to do now. Didn't know there is a git mv and also didn't know that there is something that would not lose the history.
Does anyone know how to keep the history?

**Comment by theute** [comment]
You need something like:
```
[theute@localhost notifications-backend]$ mkdir notifications-backend
[theute@localhost notifications-backend]$ git mv pom.xml src/ notifications-backend/
```
then add your pom.xml and update `build_deploy.sh` to use the new location of Dockerfile

**Comment by gwenneg** [comment]
Losing the git history would be really bad. You should probably undo all changes and start over with `git mv`.

**Pull request closed without merging**